STATUS: INLINE SUCCESSFUL
MODIFIED: com.github.kevinsawicki.http.HttpRequestTest.basicProxyAuthentication
MODIFIED FILE: /home/awshi2/kevinsawicki/http-request/lib/src/test/java/com/github/kevinsawicki/http/HttpRequestTest.java
CLEANER: com.github.kevinsawicki.http.HttpRequestTest.nullConnectionFactory
CLEANER FILE: /home/awshi2/kevinsawicki/http-request/lib/src/test/java/com/github/kevinsawicki/http/HttpRequestTest.java
POLLUTER: com.github.kevinsawicki.http.HttpRequestTest.customConnectionFactory
POLLUTER FILE: /home/awshi2/kevinsawicki/http-request/lib/src/test/java/com/github/kevinsawicki/http/HttpRequestTest.java
ORIGINAL CLEANER SIZE: 6
NEW CLEANER SIZE: 1
ELAPSED TIME: 19.332000017166138
==========================
@@ -966,0 +966,3 @@
+ {
+     HttpRequest.setConnectionFactory(null);
+ }
